### DISCUSSION TITLE
how would i make this work with teleport?

### DISCUSSION THREAD

**Discussion by eedgar** [OpenIDC/mod_oauth2#29] [Q&A]
https://goteleport.com/docs/application-access/guides/jwt/

**Comment by zandbelt** [comment]
for starters, you'l need to change the standard OAuth 2.0  header that is being read from `Authorization: bearer` to `Teleport-Jwt-Assertion` using:
```apache
OAuth2AcceptTokenIn header name=Teleport-Jwt-Assertion&type=
```
the rest should be standard from a quick read of their docs
